content/guidelines/reviewing.en.md

```diff
@@ -61,7 +61,12 @@ Your focus should be on two aspects: accuracy and quality.
 7. Verify the version of the document in the contribution tool by clicking on the _Verify version_ button.
 8. Ensure that all checks generated by the OTA-bot are manually checked.
 9. When you confirm that the term contribution has been made to the correct collection, proceed to add a review.
-10. Merge the contribution.
+
+## Merging the Pull Request
```

Comment:
This title phrasing is inconsistent with the others (here, +ing, while others are “how to…”).

I also wonder if the paragraph would not be better placed _after_ the “how to review” (and “how to make change”) sections, since one would merge after reviewing and updating 🙂

Reply:
I agree.

Reply:
I have added the `fixing out of date warning`  section right after the `making changes to ...` section. The both titles seems consistent with each other at the moment, which is why i left it as-is (I can change it if it doesn't make sense, but I also wanted to share that the two titles were written in the present continuous tense because, an action in progress is being emphasized)